src/magellan_motion/src/path_planner/planner.cpp

```diff
@@ -138,8 +144,8 @@ Path PathPlanner::plan(Point goal) {
         for (int dir = 0; dir < NUMOFDIRS; dir++) {
             numSucc++;
             // generate all valid successors
-            double newx = next->xPose + dX[dir] * .1;
-            double newy = next->yPose + dY[dir] * .1;
+            int newx = next->xPose + dX[dir];
+            int newy = next->yPose + dY[dir];
 
             double xCellsRaw = newx / _resolution;
```

Comment:
This can be removed now. You can just use `newx` and `newy` in the call to `getKey` on line 156

Reply:
Got rid of xCellsRaw, yCellsRaw , xCells, and yCells and replaced them with newx and newy in the call to getKey in the latest commits.